ubuntu/scripts/ubuntu-bootstrap.sh

```
@@ -47,6 +47,8 @@ if ! echo "$PATH" | grep -q "$HOME/.local/bin"; then
   echo 'export PATH="$HOME/.local/bin:$PATH"' >> "$HOME/.zshrc"
 fi
 
-# Print summary
+# Invoke zsh
+zsh
+
 cat <<EOF
```

Shellcheck (0.11.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)